pkg/datapath/linux/node.go

```diff
@@ -570,11 +570,22 @@ func (n *linuxNodeHandler) insertNeighbor(newNode *nodeTypes.Node, ifaceName str
 		scopedLog.WithError(err).Error("Failed to retrieve route for remote node IP")
 		return
 	}
+
+	if len(routes) == 0 {
+		scopedLog.Error("Got no routes to the remote node IP")
```

Comment:
Can we clarify this further I wonder?

```suggestion
		scopedLog.Error("Remote node IP not reachable on Layer 2. Connectivity to pods on that node may be unavailable.")
```

Reply:
I'll extend the error message as per your suggestion.